src/Utils/removeDuplicates.ts

```diff
@@ -66,15 +39,14 @@ export const groupAndMergeByColumn = (
     return mergedRow.join(",");
   });
 
-  // Return the final cleaned data as a string
-  return mergedLines.join("\n");
+  return header.concat(mergedLines.join("\n"));
 };
 
 export const preventDuplicatePatientsDuetoPolicyId = (data: string): string => {
-  const result = groupAndMergeByColumn(
+  const result = csvGroupByColumn(
     data,
-    0, // Group by Patient ID
-    [5], // Merge policy IDs
+    "Patient ID", // Group by Patient ID
+    ["policy ID"], // Merge policy IDs
```

Comment:
why do we have to specify which all columns to merge? can't we merge for all columns if there are duplicates in any except the primary column (patient id)?

Reply:
@rithviknishad I restuctured the code without passing the merge columns and merge for columns if have duplicatest